src/sentry/static/sentry/app/components/group/releaseStats.tsx

```diff
@@ -13,14 +13,16 @@ type Props = {
   environments: Environment[];
   allEnvironments: Group | undefined;
   group: Group | undefined;
+  currentRelease: any | null | undefined; // TODO(ts)
```

Comment:
any | null doesn't make much sense, but for now, you could write it as below

```suggestion
  currentRelease?: any | null  // TODO(ts)
```